internals/maintenance_scripts.py

```diff
@@ -216,3 +216,11 @@ def get_template_data(self, **kwargs):
 
     ndb.put_multi(batch)
     return f'{count} Stages entities updated of {stages.count()} available stages.'
+
+class BackfillFeatureLinks(FlaskHandler):
+  def get_template_data(self, **kwargs):
+    """Backfill feature links for existing feature entries."""
+    self.require_cron_header()
+    all_feature_entries = FeatureEntry.query().fetch()
+    count = batch_index_feature_entries(all_feature_entries, True)
```

Comment:
I think this might be too slow because `batch_index_feature_entries` does 2 NDB queries inside a loop that iterates over each FeatureEntry.    Our production server has about 2500 FeatureEntries, so that would be 5000 queries, plus whatever writes are needed.

I think a simple way to make sure that it could work would be to accept start= and num= query string parameters.  That way, if the first attempt times out, I can run with start=0&num=500 and then take it in steps.

Reply:
After talking about this, I think that we can try without star= and num= because each run will very likely make progress.